packages/website/src/components/Playground.tsx

```diff
@@ -60,133 +74,151 @@ function Playground(): React.JSX.Element {
-              <div className={styles.playgroundInfoHeader}>
-                <EditorTabs
-                  tabs={detailTabs}
-                  active={state.showAST ?? false}
-                  change={(v): void => setState({ showAST: v })}
-                />
-                {state.showAST === 'es' && (
-                  <ESQueryFilter
-                    onChange={setEsQueryFilter}
-                    onError={setEsQueryError}
-                  />
-                )}
-              </div>
-
-              {(state.showAST === 'es' && esQueryError && (
-                <ErrorViewer
-                  type="warning"
-                  title="Invalid Selector"
-                  value={esQueryError}
+            )) ||
+              (state.showAST === 'types' && astModel?.storedTsAST && (
+                <TypesDetails
+                  typeChecker={astModel.typeChecker}
+                  value={astModel.storedTsAST}
+                  onHoverNode={setSelectedRange}
+                  cursorPosition={position}
                 />
               )) ||
-                (state.showAST === 'types' && astModel?.storedTsAST && (
-                  <TypesDetails
-                    typeChecker={astModel.typeChecker}
-                    value={astModel.storedTsAST}
-                    onHoverNode={setSelectedRange}
-                    cursorPosition={position}
-                  />
-                )) ||
-                (state.showAST && astModel && (
-                  <ASTViewer
-                    key={String(state.showAST)}
-                    filter={state.showAST === 'es' ? esQueryFilter : undefined}
-                    value={
-                      state.showAST === 'ts'
-                        ? astModel.storedTsAST
-                        : state.showAST === 'scope'
-                        ? astModel.storedScope
-                        : state.showAST === 'es'
-                        ? astModel.storedAST
-                        : undefined
-                    }
-                    showTokens={state.showTokens}
-                    enableScrolling={state.scroll}
-                    cursorPosition={position}
-                    onHoverNode={setSelectedRange}
-                  />
-                )) || <ErrorsViewer value={markers} />}
-            </div>
-          </ConditionalSplitPane>
-        </ConditionalSplitPane>
-      </div>
+              (state.showAST && astModel && (
+                <ASTViewer
+                  key={String(state.showAST)}
+                  filter={state.showAST === 'es' ? esQueryFilter : undefined}
+                  value={
+                    state.showAST === 'ts'
+                      ? astModel.storedTsAST
+                      : state.showAST === 'scope'
+                      ? astModel.storedScope
+                      : state.showAST === 'es'
+                      ? astModel.storedAST
+                      : undefined
+                  }
```

Comment:
maybe this is cleaner?
```suggestion
                  value={ {
                    ts: astModel.storedTsAST,
                    scope: astModel.storedScope,
                    es: astModel.storedAST,
                  }[state.showAST] }
                  
```

Reply:
agree, https://github.com/typescript-eslint/typescript-eslint/issues/7630